docs/admin/pool_operations.md

```diff
@@ -417,12 +417,12 @@ will be decided based on the remaining ACL rules.
 
 ### Exclusion & Self-healing
 
-An operator can exclude one or more targets from a specific DAOS pool using the rank
-the target resides on as well as the target idx on that rank. If a target idx list is
-not provided then all targets on the rank will be excluded. Excluding a target will
-automatically start the rebuild process.
+An operator can exclude one or more engines or targets from a specific DAOS pool
+using the rank the target resides on as well as the target idx on that rank.
```

Comment:
...using the rank that the target resides, as well as the target idx on that rank.